x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/search_indices_logic.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LogicMounter } from '../../../__mocks__/kea_logic';
+import { searchIndices, searchEngines } from '../../__mocks__';
+
+import { SearchIndicesLogic } from './search_indices_logic';
+
+describe('SearchIndicesLogic', () => {
+  const { mount } = new LogicMounter(SearchIndicesLogic);
+
+  const DEFAULT_VALUES = {
+    searchEngines: [],
+    searchIndices: [],
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('has expected default values', () => {
+    mount();
```

Comment:
nitpick: you can move this mount to beforeEach and remove it from the specific tests.